GG/src/nanovg/stb_image.h

```diff
@@ -1375,6 +1378,8 @@ static unsigned char *stbi__convert_format(unsigned char *data, int img_n, int r
          default: STBI_ASSERT(0);
       }
       #undef CASE
+
+      #pragma GCC diagnostic push
```

Comment:
Shouldn't be there `#pragma GCC diagnostic pop` to pair up with `push` above?

Reply:
Yes it should

Reply:
Fixed via 4dd097c66614fe09845c4b4f4d535a62376e11c0